internal/server.go

```diff
@@ -12,6 +12,12 @@ import (
 	"golang.org/x/oauth2"
 )
 
+const (
+	allAuthenticated       = "system:authenticated"
```

Comment:
Can you please document what is the meaning of this variable?

Reply:
This is one of the default roles: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings